catalog/FAO/FAO_GHG_1_DROS_A.jsonnet

```diff
@@ -0,0 +1,135 @@
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'UN FAO Drained Organic Soils (Area, Annual)',
+  version: '2.0',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+			The UN FAO DRained Organic Soils Area (DROS-A) dataset disseminates annual estimates of the area of cropland and grassland organic soils that are drained 
+		for agricultural activities. This image collections consists of 2 bands:
+		  1. Drained area under cropland organic soils.
+		  2. Drained area under grazed grassland organic soils.
+			Values are expressed in hectares (ha) and are available for the period 1992–2020. The data is available globally at a 
+		resolution of 0.0083333 X 0.0083333 degrees (~1 km at the equator). This image collection is accompanied by the DROS-Emissions (DROS-E) dataset which disseminates the annual greenhouse gas (GHG) emissions that are associated with the drainage of organic soils, in a separate image collection.
+			The GHG emissions from drained organic soils dataset are computed in the Google Earth Engine (GEE) platform applying Tier I methods 
+		from the Intergovernmental Guidelines on Climate Change (IPCC) (IPCC, 2014). The FAO estimates use the distribution of histosols as a proxy for the presence of organic 
+		soils, in agreement with IPCC, and annual land cover maps as time-dependent components for yearly cropland and grassland extents. Finally, a geospatial layer of the distribution of ruminants from the FAO Gridded Livestock of the World was used to identify the grassland that is grazed.
+			Organic soils are wet soils ecosystems, characterized by high levels of organic matter, which accumulates under the anoxic conditions 
+		that exist in the presence of water. They include tropical and boreal peatlands, high-latitude bogs, ferns and mires. While organic soils 
+		cover globally a mere 3 percent of the terrestrial land area, they represent up to 30 percent of the total soil carbon, playing an 
+		important role in maintaining the earth’s carbon balance. Agriculture is a major cause of drainage of organic soils around the world and 
+		restoration of degraded organic soils is currently a priority in several countries as part of their commitments under the climate convention.
+			DROS-A and DROS-E are the basis for the country, regional and global statistics on drained organic soils that are disseminated in FAOSTAT (FAO 2020, 2022). This ‘DROS-A’ asset supersedes an earlier version, which is accessable at 'FAO/GHG/1/DROSA_A'. 
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),

```

Comment:
These are new versions of existing datasets, so the version links need to get added.  See for example ACA for how that is done.

- https://github.com/google/earthengine-catalog/blob/081a8a5eecdba0037caa0ed27c5c08e1977d9414/catalog/ACA/ACA_reef_habitat_v1_0.jsonnet#L88
- https://github.com/google/earthengine-catalog/blob/081a8a5eecdba0037caa0ed27c5c08e1977d9414/catalog/ACA/ACA_reef_habitat_v2_0.jsonnet#L97